src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -98,6 +101,16 @@ export class NavigationComponent implements OnInit, OnDestroy {
     );
   }
 
+  getFormattedClusterName(cluster: MultiCluster, shortenFsid: boolean): string {
```

Comment:
why do we need non shortenFSid?

Reply:
we need non shortenFsid for the context switcher dropdown as we are keeping full fsid in that. We show the shortened one at the context switcher's title.

![Screenshot from 2024-08-07 13-55-22](https://github.com/user-attachments/assets/a596bb24-2d27-4f6b-83cf-8dbbb7b0092a)